lms/urls.py

```diff
@@ -645,7 +645,7 @@
         r'^courses/{}/generate_user_cert'.format(
             settings.COURSE_ID_PATTERN,
         ),
-        'courseware.views.views.generate_user_cert',
+        'philu_overrides.views.generate_user_cert',
```

Comment:
put this code under same directory philu_overrides.courseware.views.views.generate_user_cert